### DISCUSSION TITLE
How to test \_\_init__?

### DISCUSSION THREAD

**Discussion by robotlightsyou** [Slow-Scallion4183/Adventure-Colussus#58] [Q&A] [answered]
Hi team,

I'm trying to get started testing what we have for init and main, but I'm having trouble importing them into my testing file. I've tried `import adventure_colossus`, `from adventure_colossus import *`, and `from adventure_colossus import __init__`, but nothing works, I get module not found errors. When I google all the responses are about excluding \_\_init__, nothing about including it. Any tips for how to structure this would be appreciated.

**Comment by wireboy5** [comment, marked as answer]
The best way I can think of to do this would to add a `__init__.py` file in the root directory and use `from .. import adventure_colussus`

Just an empty file named `__init__.py` should do

**Reply by robotlightsyou** [reply]
Thank you for this.

After adding \_\_init__.py to Adventure_Colussus/ I was unable to get `from .. import adventure_colussus` to work, so I went with `import adventure_colussus as ac` and that appears to be functioning as desired.
